instance/models/server.py

```diff
@@ -87,7 +87,8 @@ class Server(ValidateModelMixin, TimeStampedModel):
     STARTED = 'started'
     ACTIVE = 'active'
     BOOTED = 'booted'
-    PROVISIONED = 'provisioned'
+    PROVISIONING = 'provisioning'
+    ERROR = 'error'
```

Comment:
@chromano Same remark about the `ERROR` state - the information is already conveyed by `PROGRESS_FAILED`, and I don't think this is still being used by your code.